misinformation/summary.py

```diff
@@ -0,0 +1,83 @@
+from misinformation.utils import AnalysisMethod
+import torch
+from PIL import Image
+from lavis.models import load_model_and_preprocess
+
+
+class SummaryDetector(AnalysisMethod):
+    def __init__(self, subdict: dict) -> None:
+        super().__init__(subdict)
+        self.subdict.update(self.set_keys())
+        self.image_summary = {
+            "const_image_summary": None,
+            "3_non-deterministic summary": None,
+        }
+
+    summary_device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
It runs fine on my laptop with no GPU (but takes a while).

Reply:
Super!